packages/cozy-client/src/models/account.spec.js

```diff
@@ -1,8 +1,8 @@
 import MockDate from 'mockdate'
```

Comment:
This commit should be marked as Breaking Changes. with a short description of how to handle the BC :)